src/v1/Common.php

```diff
@@ -141,4 +141,31 @@ public static function validateData($dataStream, $dataFormat)
       throw new \Exception(implode(', ', $errors->all()), 400);
     }
   }
+
+  /**
+   * With token information, get list of ids of organization and sub-organization
+   * This is usefull to restrict get elements
+   */
+  public static function getOrganizations($token)
+  {
+    $listIds = [$token->organization_id];
+    if ($token->sub_organization)
+    {
+      $organization = \App\v1\Models\Item::find($token->organization_id);
+      $orgs = \App\v1\Models\Item::where('type_id', 1)->where('treepath', 'like', $organization->treepath . '%');
+      $listIds = $orgs->pluck('id')->toArray();
+    }
+    return $listIds;
+  }
+
+  /**
+   * Get the parents of the organization stored in token variable
+   */
+  public static function getParentsOrganizations($token)
```

Comment:
(nitpick) I like to have more explicit names. The methods return ids, so `getParentsOrganizationsIds` and `getOrganizationsIds`. So when someone else read the code, he knows that the methods don't return full objects.

Reply:
Renamed, very good observation ^_^